featuretools/computational_backends/feature_set_calculator.py

```diff
@@ -936,7 +939,16 @@ def update_feature_columns(feature_data, data):
         data.update(new_cols)
         return data
 
-    return data.assign(**new_cols)
```

Comment:
I assume this assign is doing the inserts under the hood that the warning originates from?

Reply:
Yeah, correct